pkg/controller/manager/manager_controller.go

```diff
@@ -331,6 +326,21 @@ func (r *ReconcileManager) Reconcile(request reconcile.Request) (reconcile.Resul
 				return reconcile.Result{}, nil
 			}
 		}
```

Comment:
This is a somewhat sidetrack issue, but do we want the above section (lines 323 - 333)? 

Since we don't support users bringing their own secret for the tunnel certificate, we don't need logic that involves checking or trying to move the secret over into `tigera-manager` namespace. Better to remove it to close the loop.

Reply:
Let's do it on a separate PR.